### DISCUSSION TITLE
Entity linking example does not run in anaconda

### DISCUSSION THREAD

**Discussion by fsasaki** [explosion/spaCy#8119] [Help: Coding & Implementations] [answered]
With the latest scripts for the NEL demo
https://github.com/explosion/projects/commit/becd8c384a98d5f0453828aa9855573ded71c1aa
I get this error
https://github.com/dmlc/xgboost/issues/1715
I am able to fix this on Windows using this solution
https://github.com/dmlc/xgboost/issues/1715#issuecomment-705998993
But I am wondering if there is another solution. Maybe the issue is just specific to anaconda.

**Comment by svlandeg** [comment]
I've never seen that error before, to be honest. When exactly is it thrown - at which line of which script? What does your environment look like?

**Comment by polm** [comment, marked as answer]
Just to expand on your links so we can search for this later, you are having an issue with multiple copies of libiomp5. You fixed it on Windows by setting the environment variable `KMP_DUPLICATE_LIB_OK` to `True`. 

I've seen reports of issues like this occasionally, though since they're not turning up in search here it might have been on Stack Overflow. We've never been able to reproduce it or figure out exactly what's going wrong :confused: . I think sometimes it has been resolved by reinstalling the whole Python environment.

What I can say is that a lot of people seem to have this issue (see [here](https://stackoverflow.com/questions/53014306/error-15-initializing-libiomp5-dylib-but-found-libiomp5-dylib-already-initial)) and it doesn't appear to be spaCy specific. Sorry I can't be of more help.

**Reply by fsasaki** [reply]
Thank you for looking into this, @polm . I understand now that this is not spaCy specific, and will work with the fix other people use.
